administration/src/util/translations/de.json

```diff
@@ -128,14 +118,35 @@
     "blueCardRequirementsTitle": "Ich erfülle folgende Voraussetzung für die Beantragung einer blauen Ehrenamtskarte:",
     "applicationStepPersonalData": "Persönliche Angaben",
     "applicationStepRequirements": "Voraussetzungen",
-    "applicationStepSubmit": "Antrag Senden"
+    "applicationStepSubmit": "Antrag Senden",
+    "title": "Bayerische Ehrenamtskarte beantragen",
+    "sentSuccessfully": "Erfolgreich gesendet",
+    "sentError": "Fehler beim Senden",
+    "nextStepButton": "Nächster Schritt",
+    "backButton": "Zurück",
+    "submitApplicationButton": "Antrag Abschicken",
+    "discardInputsTitle": "Alle Eingaben verwerfen?",
+    "discardInputsContent": "Wollen Sie wirklich alle Eingaben unwiderruflich löschen?",
+    "submitSuccessText": "Ihr Antrag für die Ehrenamtskarte wurde erfolgreich übermittelt.<br/>Über den Fortschritt Ihres Antrags werden Sie per E-Mail informiert.<br/>Sie können das Fenster jetzt schließen.",
+    "invalidInputError": "Ungültige bzw. fehlende Eingaben entdeckt. Bitte prüfen Sie die rot markierten Felder."
   },
-  "applications": {
+  "applicationsOverview": {
     "allApplications": "Alle Anträge",
     "accepted": "Akzeptiert",
     "rejected": "Abgelehnt",
     "withdrawed": "Zurückgezogen",
     "open": "Offen",
+    "status": "Status",
```

Comment:
```suggestion
```
duplicated item

Reply:
its used in different namespaces as already discussed in another conversation. If this will occur more often we can move it to misc namespace